src/Controller/QuestionTemplateController.php

```diff
@@ -83,12 +83,12 @@ public function deleteQuestion(Request $request, array $requestAttributes): Resp
     public function getQuestions(Request $request, array $requestAttributes): Response
     {
         $filters = isset($requestAttributes['text']) ? ['text' => $requestAttributes['text']] : [];
-        $count = $this->questionTemplateService->getQuestionNumber($filters);
+        $count = $this->questionTemplateService->getFilteredQuestionsNumber($filters);
```

Comment:
I personally would like the method to be called without the "filtered" part, but this is just my option

Reply:
@Raresmldvn @andreiuta97 I have no attachments on the naming I suggested if you consider it best to use another one, please do.

Reply:
no, it's ok :D I said it's just a personal preference, but I'm good with this